core/src/main/kotlin/gateway/handler/DefaultGatewayEventInterceptor.kt

```diff
@@ -36,6 +36,7 @@ public class DefaultGatewayEventInterceptor @KordPreview public constructor(
     public constructor() : this(customContextCreator = null)
 
     private val listeners = listOf(
+        UnknownEventHandler(),
```

Comment:
i ordered the other handlers alphabetically to have the same order as the files in the ide, could you put it in the right place?

Reply:
sure